sdk/spring/spring-cloud-azure-autoconfigure/src/main/java/com/azure/spring/cloud/autoconfigure/aad/properties/AadAuthenticationProperties.java

```diff
@@ -193,7 +178,7 @@ public void setApplicationType(AadApplicationType applicationType) {
         /**
          * If "true", use "v1.0/me/transitiveMemberOf" to get members. Otherwise, use "v1.0/me/memberOf".
          */
-        private Boolean useTransitiveMembers = false;
+        private boolean useTransitiveMembers = false;
```

Comment:
Why this change is necessary?

Reply:
there's no need for using a Boolean, we already set it with a default value false, and the get method for it is   
```
public boolean isUseTransitiveMembers() {
            return useTransitiveMembers;
        }
```. If it's a Boolean, chances are it will throw NPE.